src/pages/ProjectStart/Components/CardItem.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import styled from 'styled-components/macro';
+import OpenedItem from './OpenedItem';
+
+function CardItem({ title, redTitle, buttonText }) {
+  const [opened, setOpened] = useState(false);
+  const openEditor = () => {
+    setOpened(!opened);
+  };
+  return (
```

Comment:
```suggestion
function CardItem({ title, redTitle, buttonText }) {
  const [isOpened, setIsOpened] = useState(false);
  
  const toggleEditor = () => {
    setIsOpened(!isOpened);
  }
  
  return (
```
- 구분 되는 로직 끼리는 한줄 띄워주시면 좋을 것 같습니다
- 함수의 동작을 보면 toggle에 가까워서 이름을 바꾸면 어떨까 제안해봅니다!
- 변수명도 `is-` 접두사를 붙여서 boolean 타입임을 드러내는게 어떨까요?

Reply:
👌🏻👌🏻👌🏻👌🏻👌🏻